geo-search-demo/js/app.js

```diff
@@ -278,25 +294,34 @@ $(document).ready(function() {
   }
 
   function attachInfoWindow(marker, hit) {
-    var message = (hit.name === hit.city) ? hit.name+' - '+hit.country : hit.name+' - '+hit.city+' - '+hit.country;
-    var infowindow = new google.maps.InfoWindow({ content: message });
-    marker.addListener('click', function() {
-      info_window = infowindow.open(marker.get('map'), marker);
-      setTimeout(function(){infowindow.close()}, 3000);
+    var message;
+
+    if (hit.name === hit.city) {
+      message = hit.name + ' - ' + hit.country;
+    } else {
+      message = hit.name + ' - ' + hit.city + ' - ' + hit.country;
+    }
+
+    var infowindow = new google.maps.InfoWindow({content: message});
+    marker.addListener('click', function () {
+      var infoWindow = infowindow.open(marker.get('map'), marker);
```

Comment:
Yeah, I was curious about that. It was a global var before, so I'm not sure if it's needed anywhere. @kokliKo thoughts?